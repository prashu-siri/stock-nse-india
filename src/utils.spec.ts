import { getDataSchema } from './utils'
describe('Utils', () => {
    test('getDataSchema for date value having typeStrict:false',()=>{
        const data={
            date: new Date('2022-09-12')
        }
        expect(getDataSchema(data,false)).toMatchSnapshot()
    })
})