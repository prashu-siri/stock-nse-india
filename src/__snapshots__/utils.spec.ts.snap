// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Utils getDataSchema for date value having typeStrict:false 1`] = `
Array [
  "date: any",
]
`;
